test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -229,7 +229,7 @@ def test_framework_version_cpu(image):
         if tested_framework == "autogluon.core":
             versions_map = {
                 # container version -> autogluon version
-                '0.3.2': '0.3.1',
+                # '0.3.2': '0.3.1',
```

Comment:
does this need to be removed?

Reply:
Yes, they are no longer supported.

Reply:
@gradientsky Can you confirm this?

Reply:
I left this in place as an example: just in case if this functionality would be required in future.